renovate.json

```diff
@@ -2,9 +2,9 @@
   "extends": [
     "config:base",
```

Comment:
By keeping `config:base` here, won't we continue setting `group:monorepos`?  Is that intended?

Reply:
I have no intention actually, but I also think it won't cause any problems (not 100% sure though).